golos.publication/parameters.hpp

```diff
@@ -58,10 +58,24 @@ namespace golos {
         }
     };
     using referral_acc_prm = param_wrapper<st_referral_acc, 1>;
+    
+    struct st_curators_prcnt : parameter {
+        uint16_t min_curators_prcnt;
+        uint16_t max_curators_prcnt;
+
+        void validate() const override {
+            eosio_assert(min_curators_prcnt > 0 && min_curators_prcnt < 100,
```

Comment:
so percent here is integer between 0 and 100? in all other places we have fixed point with 2 fractional digits (0-10000). propose either unify percent and make if fixed point or change type to `uint8_t`, so it's more clear it's integer. also add comment if using integer percent